tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Tests/Tests/ActionTests/Action_BeginDialogWithExpr.test.dialog

```diff
@@ -0,0 +1,84 @@
+{
+    "$schema": "../../../tests.schema",
+    "$kind": "Microsoft.Test.Script",
+    "dialog": {
+        "$kind": "Microsoft.AdaptiveDialog",
+      "id": "planningTest",
+      "dialogs":  [ "AskNameDialog", "TellJokeDialog" ],
```

Comment:
>"AskNameDialog", "TellJokeDialog"  [](start = 20, length = 34)

like any dialogs these can be defined in line or be the name of an id.  Please make sure to unit test both scenarios (aka, put TellJokeDialog" in line side by side with AskNameDialog)

Reply:
Added support of array of JObjects and array of mixed string and JObject